lib/parser.js

```diff
@@ -350,6 +350,27 @@ const args = [
     help: '(IOS-only) Local port used for communication with ios-webkit-debug-proxy'
   }],
 
+  [['--remote-debug-proxy-port'], {
+    defaultValue: false,
+    dest: 'remoteDebugProxyPort',
+    required: false,
+    type: 'int',
+    example: "12000",
+    help: '(IOS-only, <= 11.2) If set, Appium sends and receives remote debugging messages' +
```

Comment:
As implemented in the `appium-remote-debugger`, this will only work for simulators. That should be mentioned here and below.

Reply:
Thanks for the feedback! I wasn't aware that desired capabilities just work out of the box. I removed the server flags and added them as desired capabilities.